app/services/oneoffs/npq/bulk_change_applications_to_pending.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Oneoffs::NPQ
+  class BulkChangeApplicationsToPending
+    attr_reader :npq_application_ids
+
+    def initialize(npq_application_ids:)
+      @npq_application_ids = npq_application_ids
+    end
+
+    def run!(dry_run: true)
+      result = {}
+
+      ActiveRecord::Base.transaction do
+        result = npq_application_ids.each_with_object({}) do |npq_application_id, hash|
+          npq_application = NPQApplication.find_by(id: npq_application_id)
```

Comment:
We should raise an error if the application doesn't exist, which could indicate the CSV data is incorrect.